caikit/core/data_model/dataobject.py

```diff
@@ -30,6 +30,7 @@
     Union,
     get_args,
     get_origin,
+    TypeVar,
```

Comment:
NIT: I think this will fail `fmt`

Reply:
This is fixed, or at least `tox fmt` passes on my machine.